### PR TITLE
Some Implementations done

### PR DESCRIPTION
**Pull request by K-Kumar-01**
Well i added some code for profile cards.
See and let me know

### CODE REVIEW

File: Cards/ProfileDisplay/index.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
+html{
+  background-color: gray;
+}
+.profile-block{
+  width:25%;
```
Comment (king-11) [inline Cards/ProfileDisplay/index.css]: It looks a bit stretched try going with 25rem.

File: Cards/ProfileDisplay/index.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  margin:100px auto 0;
+  text-align: center;
+  background-color: #f2f2f2;
+  border-radius: 10px;
+  padding:10px;
+}
+
+.profile-screen{
+  transform: translateY(-5%);
```
Comment (king-11) [inline Cards/ProfileDisplay/index.css:14]: Looks good.

File: Cards/ProfileDisplay/index.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+.profile-basic-intro{
+  font-size:1.5rem;
+  font-weight: bold;
+  color:#676767;
+}
+
+.profile-contact-info{
+  display:flex;
+  width:80%;
+  margin: 0 auto;
```
Comment (king-11) [inline Cards/ProfileDisplay/index.css]: Add a bit of top margin would look better.

File: Cards/ProfileDisplay/index.css [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  justify-content: space-around;
+}
+.profile-contact-info a{
+  text-decoration: none;
+  color:#121212;
+}
+
+.profile-contact-info a:hover{
+  transform: scale(1.2);
```
Comment (king-11) [inline Cards/ProfileDisplay/index.css:51]: Nice :). Another thing we can do is add the filter:grayscale property to icons which you can find over here https://icons8.com/icons . 
Please consider adding them to as would look very nice then.

File: Cards/Profiles/index.css [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  font-size: 2.5rem;
+  font-weight: bold;
+  padding-top: 20px;
+  text-shadow: 1px 1px black;
+}
+
+.profile-work{
+  font-size:1.5rem;
+}
+.upper-profile-container img{
```
Comment (king-11) [inline Cards/Profiles/index.css:27]: Can be a bit larger in size as isn't quite prominent right now.

File: Cards/Profiles/index.css [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+  color: white;
+  font-size: 30px;
+  font-weight: 700;
+  padding: 5px;
+  font-family: sans-serif;
+  text-shadow: 1px 1px 4px #000000;
+  text-decoration: none;
+  display: inline-block;
+  cursor: pointer;
+}
```
Comment (king-11) [inline Cards/Profiles/index.css]: Good Functionality for artists needs a bit of reduction in size. Adding two distinct buttons one for Insta and other for Facebook can be a better addon.

File: Cards/Profiles/index.css [hunk 7]
```diff
@@ -0,0 +1,95 @@
+.profile-block-container{
+  width:500px;
+  margin: 0 auto;
+}
```
Comment (king-11) [inline Cards/Profiles/index.css]: Change the width to 25 rem. Also adding box-shadow will create a nice 3d feel.

File: Cards/Profiles/index.css [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+  color: #aaaaaa;
+  font-family: cursive;
+  justify-content: space-around;
+  text-align: left;
+}
+
+.headings div{
+  padding: 5px 68px 5px;
+  border-bottom: 3px solid #aaaaaa;
+}
```
Comment (king-11) [inline Cards/Profiles/index.css]: Font size needs to be increased isn't visible on mobile screen.

**Review by king-11 — CHANGES_REQUESTED** [review]

**Comment by K-Kumar-01** [comment]
I have opened another pull request with the requested changes done.

File: Cards/Profiles/index.css (hunk 6 above)
Comment (K-Kumar-01) [reply Cards/Profiles/index.css]: Buttons will be distinguished by bg colors.
Green one was for fb. as there the background was already blue.
Newly added will be of insta with background as purple-pink

**Pull request closed without merging**